crates/merkle-tree/src/class.rs

```diff
@@ -113,3 +114,70 @@ impl crate::storage::Storage for ClassStorage<'_> {
         Ok(leaf)
     }
 }
+
+/// Revert all class trie updates.
+pub fn revert_class_updates(
```

Comment:
The forward equivalent lives in the pathfinder crate I think. Did you add these here to have access for the example?